attestation/link/link.go

```diff
@@ -74,7 +74,7 @@ func WithExport(export bool) Option {
 }
 
 type Link struct {
-	PbLink   v0.Link
+	PbLink   v0.Link `jsonschema:"title=Link,description=In-toto link attestation containing materials and products"`
```

Comment:
Do you know how this render down into the contents of the protobufs?

Reply:
That is a great question